css/pages/index.css

```diff
@@ -3,50 +3,122 @@
 }
 
 .container {
-  width: 750px;
+  width: 100%;
   text-align: center;
   font-size: 1.5rem;
 }
 
+.second-screen {
+  width: 100%;
+}
+
 .cards-title {
-  margin-bottom: 32px;
-  color: #e20665;
+  margin-top: 1041px;
+  margin-bottom: 60px;
+  width: 100%;
+}
+
+h3 {
+  color: #091133;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  letter-spacing: 10%;
+  font-weight: 600;
+  font-size: 250%;
+  line-height: 66%;
+  font-style: normal;
+  font-family: 'raleway';
 }
 
 .cards {
   display: flex;
-  justify-content: space-around;
+  justify-content: center;
+  width: 100%;
+  margin-left: 5%;
 }
 
 .card {
   border: 1px solid #f3efef;
-  box-shadow: 0 0 15px -9px rgba(0, 0, 0, 0.65);
-  border-radius: 2px;
-  width: 30%;
-  padding: 18px 15px;
+  border-radius: 18px;
```

Comment:
use relative unit for defining borders

Reply:
If someone changes their system font size to 32px (twice of default 16px) and if the border-radius is 1rem instead of 16px (let's say), then their border-radius will be 32px instead of 16px, making it more rounded, taking it closer to a circular shape, resulting in breaking design. What do you think @ghosharnab969?

Reply:
> If someone changes their system font size to 32px (twice of default 16px) and if the border-radius is 1rem instead of 16px (let's say), then their border-radius will be 32px instead of 16px, making it more rounded, taking it closer to a circular shape, resulting in breaking design. What do you think @ghosharnab969?

No, the changes won't occur as you said. The shape of the border actually remains preserved if at all the font-size of the  root element is changed.

Reply:
Can you please provide any reference to what you're saying? @ghosharnab969

Reply:
ya sure! Hope this reference helps.
[https://stackoverflow.com/questions/23991054/does-rem-make-sense-on-border-radius](url)

Reply:
ya sure! hope this reference helps. @swarajpure 
[https://stackoverflow.com/questions/23991054/does-rem-make-sense-on-border-radius](url)